seed/challenges/02-javascript-algorithms-and-data-structures/debugging.json

```diff
@@ -68,6 +68,42 @@
       "translations": {}
     },
     {
+      "id": "587d7b83367417b2b2512b37",
+      "title": "Clarifying the In-Browser Console Log",
```

Comment:
I think the title should be something very obvious, like "Understand How the freeCodeCamp Console is Different than the Browser Console"

Reply:
Oh I think I see what you are referring to @HKuz. the freeCodeCamp Console in this case is the CodeMirror block that is part of the html?  Ok I think I see what you are getting at...

Reply:
On second thought maybe I am misunderstanding, going to have to run it and see...

Reply:
Was a bit confused here.  There are actually three, maybe four different things happening here, one is the explanation of the CodeMirror console and that functionality that exists inside of that, another is the explanation of the console.logs inside the CodeMirror text-editor block and how that output functions in terms of the running of the code and the print of those outputs in the Browser console.  I was only addressing one of these angles not both...